app/helpers/onboardings_helper.rb

```diff
@@ -0,0 +1,2 @@
+module OnboardingsHelper
```

Comment:
Is this mistakenly included?

Reply:
Yep, leftovers from another branch snuck in here. 🧹  920e491b